src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -74,15 +74,15 @@ public long RawContentLength
         /// </summary>
         public int StatusCode
         {
-            get { return WebResponseHelper.GetStatusCode(BaseResponse); }
+            get => WebResponseHelper.GetStatusCode(BaseResponse);
```

Comment:
We could use expression-bodied members for these properties.

Reply:
Done